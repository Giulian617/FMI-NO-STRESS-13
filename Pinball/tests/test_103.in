100 1
84 86 3
42 84 10
53 84 5
89 84 7
6 84 3
25 84 6
2 84 3
81 84 7
84 66 4
93 84 9
35 84 3
84 52 4
10 84 1
37 84 1
84 36 3
30 84 8
54 84 5
84 90 7
32 84 7
72 84 2
87 84 10
97 84 7
92 84 2
84 48 5
84 3 3
84 77 2
84 40 2
33 84 6
84 85 8
57 84 4
60 84 7
84 28 4
22 84 1
58 84 3
21 84 9
84 9 6
84 26 3
84 12 7
84 4 9
84 18 5
84 59 7
84 64 5
84 79 2
91 84 8
95 84 10
46 84 1
84 34 5
84 68 2
84 14 10
1 84 10
84 67 8
100 84 8
16 84 2
84 61 5
84 24 4
44 84 10
56 84 3
84 69 9
84 74 5
84 83 1
84 50 2
73 84 4
29 84 10
71 84 3
84 96 5
84 75 8
17 84 10
43 84 3
84 55 8
84 23 9
84 82 3
84 80 10
84 62 10
88 84 10
84 20 6
19 84 7
38 84 7
63 84 4
84 13 4
84 5 5
84 7 9
39 84 8
84 41 7
84 70 10
84 49 4
15 84 7
84 27 7
99 84 6
84 47 9
76 84 10
84 11 7
45 84 2
84 78 2
84 51 10
84 94 10
84 65 10
31 84 10
84 8 8
84 98 8
