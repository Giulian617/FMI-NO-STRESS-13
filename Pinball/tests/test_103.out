353391066
306310108
306310108
405688441
933943872
306310108
405688441
705502499
485376774
95678172
155002878
155002878
462425775
24503718
155002878
610739863
24503718
933943872
155002878
485376774
405688441
95678172
405688441
462425775
485376774
306310108
155002878
462425775
24503718
705502499
24503718
155002878
485376774
933943872
306310108
306310108
95678172
155002878
705502499
610739863
155002878
24503718
306310108
24503718
610739863
95678172
405688441
933943872
462425775
610739863
24503718
462425775
933943872
933943872
705502499
306310108
462425775
306310108
155002878
155002878
933943872
24503718
462425775
933943872
24503718
462425775
705502499
610739863
405688441
24503718
306310108
610739863
462425775
933943872
705502499
24503718
610739863
610739863
610739863
24503718
155002878
306310108
95678172
705502499
306310108
24503718
24503718
155002878
155002878
705502499
610739863
405688441
24503718
24503718
933943872
155002878
705502499
485376774
705502499
